### DISCUSSION TITLE
command "ci(" doesn't work without being in the bracket

### DISCUSSION THREAD

**Discussion by ayoola-moore** [VSCodeVim/Vim#7801] [General]
When I type `ci(` I would expect vim jumps to the next "(" and go into insert mode. 

However, in vscodevim, I have to be within the () to make that change. That is, first jump to ( with `f(` and then enter `ci(` for insertion.  Is this a settings thing or a bug?

**Comment by wgr45097** [comment]
this seems to be a replicate of #7346

**Reply by kosekidev** [reply]
Yup.
But will it be implemented ? It works with ci" but not ci( 🤔
